python/python-hello-world/README.md

```diff
@@ -7,129 +7,48 @@
-- A new file will open populated with the standard output of the hello-world container
-![view logs](./img/logs.gif)
-
-### Debug Your Code
-- ![Debug View](./img/debug_view.png) Open [Debug view](https://code.visualstudio.com/Docs/editor/debugging)
-- Select "hello-world" as the debug configuration in the dropdown menu
-- Press the debug start button to attach a debugger session to the hello-world application
-  - Cloud Code usses the .vscode/launch.json file to locate the pods to attach a debug session to
-  - the hello-world container was built to run `ptvsd` to support debugging
-- You should see the [Debug toolbar](https://code.visualstudio.com/Docs/editor/debugging#_debug-actions) with familiar debug controls
-- You can click in the margin in `app.py` to add a breakpoint. When you send a new request to your application, it will pause at the desired line
-- You can inspect variables and stack info in the [Debug Side Bar](https://code.visualstudio.com/Docs/editor/debugging)
-- You can interact with the debugging session using the [Debug Console](https://code.visualstudio.com/Docs/editor/debugging#_debug-console-repl) in the bottom pane
-![debugger](./img/debug.gif)
-
-### Open a Terminal in Your Container
-By opening a terminal, you can monitor and debug running processes as though they were running on your local machine
-- Open the [Command Palette](https://code.visualstudio.com/docs/getstarted/userinterface#_command-palette)
-- Type "CloudCode: Get Terminal"
-- Select the pod you wish to connect to
-- You now have access to a shell inside the running container
-![container terminal](./img/term.gif)
-
-### Tasks
-This sample also includes a number of [Visual Studio Code Tasks](https://code.visualstudio.com/docs/editor/tasks) to launch common commands
-
-#### Run the Pylint Task
-Run a Python [linter](https://en.wikipedia.org/wiki/Lint_(software)) to analyze your code for issues
-- [Install Pylint](https://www.pylint.org/#install) if necessary
-- Open the [Command Palette](https://code.visualstudio.com/docs/getstarted/userinterface#_command-palette)
-- Type "Tasks: Run Task"
-- Select the "Pylint" task
-- An output window will appear showing the Pylint output
-![lint task](./img/lint.gif)
+This sample was written to demonstrate how to use the Cloud Code extension for Visual Studio code.
+* [Install Cloud Code for VS Code](https://cloud.google.com/code/docs/vscode/install)
+* [Creating a new app](https://cloud.google.com/code/docs/vscode/creating-an-application)
+* [Editing YAML files](https://cloud.google.com/code/docs/vscode/yaml-editing)
+
 
 ---
 
 ## Using the Command Line
 As an alternative to using the Cloud Code extension, the application can be deployed to a cluster using standard command line tools
 
-### Using Skaffold
+### Skaffold
 [Skaffold](https://github.com/GoogleContainerTools/skaffold) is a command line tool that can be used to build, push, and deploy your container images as you work
```

Comment:
this says "as you work" but the cmd below is not "skaffold dev". What's the value of documenting skaffold here?

Reply:
In case folks land on these templates via github and prefer to use the code as a sample without vscode or intellij

Reply:
still sounds inaccurate. maybe change command to "skaffold dev" below, or edit the descriptio.

IMO folks landing on these templates without using vscode/intellij is a very very small percentage that's not worth optimizing for.

these extra sections are probably making more confusing for the main audience to begin with.